tabcmd/locales/en/tabcmd_messages_en.properties

```diff
@@ -165,6 +165,7 @@ export.errors.need_country_and_languge=The options --country and --language must
 export.errors.white_space_workbook_view=The name of the workbook or view to export cannot include spaces. Use the normalized name of the workbook or view as it appears in the URL.
 export.errors.requires_workbook_view_name=The ''{0}'' command requires a <workbook>/<view> name
 export.errors.requires_workbook_view_param=The ''{0}'' command requires a <workbook>/<view> parameter, and there must be at least one slash (/) in this parameter
+export.errors.requires_valid_custom_view_uuid=The URL for custom views must contain a valid custom view luid
```

Comment:
You should add this English string to all the locales, so that there is at least a message while we wait for translation.